src/NoFrixion.MoneyMoov/Models/Payouts/Payout.cs

```diff
@@ -308,7 +308,9 @@ public string GetApprovalHash()
                 Currency +
                 Math.Round(Amount, 2).ToString() +
                 Destination.GetApprovalHash() +
-                Status.ToString();
+                Status.ToString() +
+                Scheduled.ToString() +
+                ScheduleDate.ToString();

```

Comment:
Might be worth using ToString("o") for the datetime string. Just in case an external system ever needs to generate/verify the hash and wants a known format.